project.clj

```diff
@@ -40,8 +41,8 @@
                  ;; we have documentation that shows how users can use it to
                  ;; send their logs to logstash, so we include it in the jar.
                  ;; we may use it directly in the future
-                 [net.logstash.logback/logstash-logback-encoder "4.5.1"]
-
+                 [net.logstash.logback/logstash-logback-encoder "4.5.1" :exclusions [ch.qos.logback/logback-access
+                                                                                     ch.qos.logback/logback-core]]
```

Comment:
Just curious why we did this as an exclusion as opposed to putting an explicit logback version dependency in the "conflict resolution" section.